### PR TITLE
Login routing fixed

### PR DESCRIPTION
**Pull request by IngeDW**
Hi, I've changed the login routes to the correct router URLs.

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 <%- include('./partials/head') %>
 
-
+<!-- Inge -->
 <body data-page="banner-home">
 
-  <form action="/" method="post" data-page="form">
+  <form action="/signup" method="post" data-page="form">
```
Comment (MarvinMichel) [inline views/index.ejs:9]: Check the action a second time. If I'm not mistaken, this should be just '/'. The form posts data to the same page. This post should handle the info and redirect the user to the new page. This is not a task for the form, but the post method.

File: views/login.ejs [hunk 2]
```diff
@@ -3,10 +3,10 @@
 
 <%- include('./partials/head') %>
 
+<!-- Inge -->
 <body data-page="banner-home">
 
-  <!-- Made by Inge -->
-  <form action="/log-in" method="post" data-page="form">
+  <form action="/log-in-feed" method="post" data-page="form">
```
Comment (MarvinMichel) [inline views/login.ejs:9]: The url from this action is non-existing. Test if the actions work as well, it probably shouldn't.

File: views/signup.ejs [hunk 3]
```diff
@@ -3,9 +3,9 @@
 
 <%- include('./partials/head') %>
 
+<!-- Inge -->
 <body data-page="banner-home">
 
-  <!-- Made by Inge -->
   <form action="/feed" method="post" data-page="form">
```
Comment (MarvinMichel) [inline views/signup.ejs:9]: Same as the others.

**Comment by IngeDW** [comment]
I understand. The form actions are now updated. Can you review the code again?

**Comment by IngeDW** [comment]
Code looks correct! I've changes the last "/" in index.ejs to an empty  " ", so the code is more consistent. Furthermore, it's correct and it kan merge with feed!

**Pull request merged**